example/src/Examples/Matrix/Symbol.tsx

```diff
@@ -9,7 +9,7 @@ const { width, height } = Dimensions.get("window");
 export const COLS = 8;
 export const ROWS = 15;
 export const SYMBOL = { width: width / COLS, height: height / ROWS };
-const symbols = "abcdefghijklmnopqrstuvwxyz".split("");
+const symbols = "abcdefghijklmnopqrstuvwxyz".toUpperCase().split("");
```

Comment:
It will be amazing to have the entire Latin extended characters just for reference, what do you think?

Both lowercase and uppercase:
`ĀāĂăĄąĆćĈĉĊċČčĎďĐđĒēĔĕĖėĘęĚěĜĝĞğĠġĢģĤĥĦħĨĩĪīĬĭĮįİıĲĳĴĵĶķĸĹĺĻļĽľĿŀŁłŃńŅņŇňŊŋŌōŎŏŐőŒœŔŕŖŗŘřŚśŜŝŞşŠšŢţŤťŦŧŨũŪūŬŭŮůŰűŲųŴŵŶŷŸŹźŻżŽž`
Only lowercase: 
`āăąćĉċčďđēĕėęěĝğġģĥħĩīĭįi̇ıĳĵķĸĺļľŀłńņňŋōŏőœŕŗřśŝşšţťŧũūŭůűųŵŷÿźżž`

[Source](https://www.piliapp.com/symbol/latin-extended/)

Reply:
Indeed this needs to be tested. In this particular example, we use a custom font that only supports these codepoints.